hat/conf/she.conf

```diff
@@ -33,15 +33,6 @@ she {
       endpoint = "insights/emotions"
       experimental = false
     }
-    {
```

Comment:
removed?

Reply:
Yes. It's experimental => only in sandbox and not in production. 
No one has used it.
The SHE function was failing anyway.